rule/var_naming.go

```diff
@@ -176,6 +135,51 @@ func (*VarNamingRule) Name() string {
 	return "var-naming"
 }
 
+func (r *VarNamingRule) applyPackageCheckRules(file *lint.File, onFailure func(failure lint.Failure)) {
+	fileDir := path.Dir(file.Name)
```

Comment:
I guess, should be `filepath` instead of `path`:
```suggestion
	fileDir := filepath.Dir(file.Name)
```